### DISCUSSION TITLE
add ability to embed video

### DISCUSSION THREAD

**Discussion by MattSidney** [TandoorRecipes/recipes#2223] [Ideas]
I love the ability to upload and add images to Tandoor recipes. 
Sometimes this is enough, along with step descriptions to show the process of something. 
However, it is also sometimes helpful to view a video of a process. At the moment I have been adding a step a the end of recipies called "video" and uploading video file to that step which then can be downloaded, as a zip (is this automated), extracted and viewed. 

The proposal here is to add the ability to attach a video file (either to a step, or as a separate entity), then allow for playback within Tandoor.

**Comment by vabene1111** [comment]
Hi, thanks for the feedback.

This is a good idea and has been planned for a bit.

The zipping before downloading is to prevent stored XSS attacks, maybe this can be fixed better in the future but I fear this might be it for now.

**Reply by MattSidney** [reply]
Ah, good to hear, love your work on this software :) 

Do you track future features in a list somewhere or just in these discussions/issue tickets?

**Reply by vabene1111** [reply]
issues are the main way we track things, i have this palanned as part of some other stuff so nothing more you need to do, just no idea when i will get around to doing this.

**Reply by Viable8237** [reply]
@vabene1111 Any updates on this feature? I would also like to see the ability to upload videos to recipes.

**Reply by fonzie24** [reply]
Same here. Would love to know if there is any movement on adding videos to recipes. My typical use-case would be to download a video recipe from Instagram and embed it into Tandoor recipes. Thanks!

**Reply by Uroth** [reply]
Yes, I would love this feature. Would be perfect to add small 10-20 second videos to each recipe so you can see roughly how its made and expected result. :)
